Source/WebInspectorUI/UserInterface/Controllers/ConsoleManager.js

```diff
@@ -293,6 +297,17 @@ WI.ConsoleManager = class ConsoleManager extends WI.Object
         WI.objectStores.consoleSnippets.putObject(snippet);
     }
 
+    _collectFailedSourceMapConsoleMessage(message)
```

Comment:
NIT: can you move this to right after `_clearMessages` so that it's not in the middle of all the event handlers?